apps/web/src/components/Common/Providers/PreferencesProvider.tsx

```diff
@@ -40,17 +40,21 @@ const PreferencesProvider: FC = () => {
     } catch {
     } finally {
       setLoadingPreferences(false);
+      return false;
     }
   };
 
   useQuery(['preferences', profileId], () => fetchPreferences());
 
   const fetchVerifiedMembers = async () => {
+    let _data;
     try {
       const response = await axios.get(`${PREFERENCES_WORKER_URL}/verified`);
       const { data } = response;
+      _data = data;
       setVerifiedMembers(data.result || []);
     } catch {}
+    return _data ? _data.result : [];
```

Comment:
rather that doing this. You can choose to return data in try?  and empty array at end as default case?

Reply:
pushed